### ISSUE TITLE
[stack] cover/fill option in images in stacks

### ISSUE THREAD

**Issue by zhenyasav** [dxos/dxos#4886]
- Chrome for stack sections

**Comment by thure** [comment]
Type-specific surfaces such as this are currently blocked by ongoing Collections work.

**Comment by wittjosiah** [comment]
@thure what makes you say this is blocked by the collections work? I think this could likely be accomplished ahead of that using just the app framework plugin provides:
```ts
export type StackProvides = {
  stack: {
    creators?: StackSectionAction[];
    actions?: (object: Stack.Section['object']) => StackSectionAction[];
  };
};
```

**Comment by thure** [comment]
That is a good way plugins can define actions, but  then the way actions are finally rendered into the stack section’s rail or options menu is to be determined.

I think the way this is done depends on whether titles and actions are brought with, or queried by, a section’s object (so the UI component has enough to work with from the Tile), or if the UI components do somehow need to expose more subcomponents so plugins can fill things in with greater granularity. The latter approach might work today, but we might not want to keep that implementation long enough to merit its cost.

**Comment by wittjosiah** [comment]
Ah I see, so the reason you're suggesting that this is related to collections is along the same lines of what we were talking about with the collapsed state? Basically, the way to pass this stuff through to the section tile means consolidating the things into a single schema for a section.

**Comment by thure** [comment]
Exactly, I think something we’d want to look at is whether/how we can enable the UI to know where to get what it needs via schemas, or if it should remain agnostic. I’m hoping we can find a solution for the former.
